src/data/example.js

```diff
@@ -0,0 +1,3 @@
+const level100Array = [{name: "north"},{name: "night"},{name: "dead"}]
+
+const episodeCodes =["S02E01", "S02E02", "S02E03"]
```

Comment:
i suggest removing this file, example.js, from your project.  It doesn't seem to serve a purpose.  You could put an entry in `.gitignore` to have git ignore it, if you wanted to keep it around only locally for the purposes of experimentation.